tests/integration_tests/build/test_licenses.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests ensuring codebase contains neccessary licenses."""
+
+import os
+
+import pytest
+
+AMAZON_COPYRIGHT = ("Copyright 2018 Amazon.com, Inc. or its affiliates. "
```

Comment:
nit: The other Python code in the repo has a different line wrap style, namely:

```python
AMAZON_COPYRIGHT = (
    "Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved."
)
```

The idea is that it keeps code easy to read, and well aligned when you do things like grep on the code base to look for something. This applies throughout, or example:

```python
            has_chromium_copyright = (
                CHROMIUM_COPYRIGHT in copy and
                CHROMIUM_LICENSE in local_license
            )
```

Reply:
Sounds great thank you! I've incorporated your feedback in the latest commit. Should I squash the commits, or will you squash then merge the PR?